doc/source/reference/c-api/dtype.rst

```diff
@@ -348,13 +352,21 @@ to the front of the integer name.
 
     64-bit unsigned integer
 
-.. c:type:: npy_(u)long
+.. c:type:: npy_long
 
-    (unsigned) long int
+    long int
 
-.. c:type:: npy_(u)longlong
+.. c:type:: npy_ulong
 
-    (unsigned long long int)
+    unsigned long int
+
+.. c:type:: npy_longlong
+
+    unsigned long long int
```

Comment:
You've swapped this entry and the one below

Reply:
Fixed.